assets/src/edit-story/app/api/apiProvider.js

```diff
@@ -105,10 +105,10 @@ function APIProvider({ children }) {
   );
 
   const getMedia = useCallback(
-    ({ mediaType, searchTerm }) => {
+    ({ mediaType, searchTerm, page }) => {
       let apiPath = media;
-      const perPage = 100;
-      apiPath = addQueryArgs(apiPath, { per_page: perPage });
+      const perPage = 20;
```

Comment:
Why change to 20? `The library will be paginated (100 per page)` The AC says https://github.com/google/web-stories-wp/issues/23

Reply:
Maybe my note was not clear: it's WIP/for easier testing.

Reply:
If it is a WIP in process, put [WIP] in title of PR and set the PR as a draft. Otherwise it will get reviewed and merged.